typings/internal-ambient.d.ts

```diff
@@ -133,7 +133,8 @@ declare namespace NodeJS {
 
   interface WebFrameMainBinding {
     WebFrameMain: typeof Electron.WebFrameMain;
-    fromId(processId: number, routingId: number): Electron.WebFrameMain;
+    fromId(processId: number, routingId: number): Electron.WebFrameMain | null;
+    fromFrameToken(processId: number, frameToken: string): Electron.WebFrameMain | null;
```

Comment:
Seems like this is typed as `| null` here, but in `web-frame-main.md` it's `| undefined`?